common/spdat.cpp

```diff
@@ -1456,41 +1456,42 @@ bool IsCompleteHealSpell(uint16 spell_id)
 }
 
 bool IsFastHealSpell(uint16 spell_id) {
-    spell_id = (
-        IsEffectInSpell(spell_id, SE_CurrentHP) ?
-        spell_id :
-        GetSpellTriggerSpellID(spell_id, SE_CurrentHP)
-    );
-
-    if (!spell_id) {
-        spell_id = (
-            IsEffectInSpell(spell_id, SE_CurrentHPOnce) ?
-            spell_id :
-            GetSpellTriggerSpellID(spell_id, SE_CurrentHPOnce)
-        );
-    }
+	spell_id = (
+		IsEffectInSpell(spell_id, SE_CurrentHP) ?
```

Comment:
This block is confusing, are we modifying the supplied parameter here?

Reply:
I thought the same, it's a catch for triggered spells to grab the original spell it seems